### PR TITLE
MD Seafood App - Pull Request

### PR DESCRIPTION
**Pull request by AmaduKamara**
# 👋 Micronaut Reviewer 

Pull Request changes fixed as requested by the previous code reviewer. 👍

## MD Seafood web app 

>In this project [Amkam](https://github.com/AmaduKamara) and [Debas](https://github.com/Debas-31) implemented:

- [x] JavaScript to make websites dynamic and build basic single-page apps.
- [x] ES6 syntax.
- [x] ES6 modules.
- [x] Callbacks and promises.
- [x] Webpack.
- [x] JavaScript best practices and language style guides in code.
- [x] AAA pattern for unit tests.
- [x] Follow Gitflow.
- [x] Solve simple git conflicts.
- [x] Send and receive data from an API.
- [x] The API documentation.
- [x] Using [Meals DB](https://www.themealdb.com/api.php) and [Microverse Involvement](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) API to display the Seafoods and record the user interactions (likes and comments).
- [x] Worked with JSON.
- [x] Perform a code review for a team member.
- [x] Use async and await JavaScript features to consume the API.
- [x] styling using Tailwind CSS framework

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,109 @@
...
+- Webpack-5
+- Jest
+
+## Live Demo
+
+[Live Demo](https://seafoods-app.netlify.app/)
+
+## Video Documentation
+[Video](https://drive.google.com/file/d/1yJMniajNASvKvGaUk6lF7yoetIP4ZV-C/view?usp=sharing)
+
```
Comment (Dipeshtwis) [inline README.md]: - [x] You have really done a great job on the presentation but one thing missing. Please highlight a piece of the code you find interesting or are proud of making.

File: README.md [hunk 2]
```diff
@@ -1 +1,109 @@
...
+```
+
+- Install Dependencies
+
+```bash
+npm install
+```
+
+To run StyleLint by itself, you may run the lint task:
+
```
Comment (Dipeshtwis) [inline README.md]: - [x] Please provide the proper steps to run the project locally. After cloning there should be a command to enter into the folder and after installing the npm package, there should be a command to run the webpack server.

```
git clone git@github.com:AmaduKamara/seafoods-app

cd seafoods-app

npm i or npm install

// To run the server
npm start

```

Also, provide the command to test the project.

**`npm test`**

File: src/commentModal.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+              class="rounded-lg shadow-lg"
+              width="350"
+            />
+          </div>
+          <div class="mt-2">
+            <div class=" my-1">
+              <h3 class="text-center text-xl md:text-3xl text-orange-500 mt-5 mb-2">${food.strMeal}</h3>
+            </div>
+          </div>
+        </div>
```
Comment (Dipeshtwis) [inline src/commentModal.js]: - [x] In the comment popup, you need to show details about the selected item like ingredients, or anything which you can fetch from the API.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @AmaduKamara & @Debas-31, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Your application is working fine.
:+1: Deployed on netlify. :heavy_check_mark:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (AmaduKamara) [reply README.md]: Yeah sure. Thanks, @Dipeshtwis for picking that up. 🤝

File: README.md (hunk 1 above)
Comment (AmaduKamara) [reply README.md]: Oooh, ok, bro. We'll definitely redo and mention that. 👍

File: src/commentModal.js (hunk 3 above)
Comment (AmaduKamara) [reply src/commentModal.js]: Ok @Dipeshtwis , We've successfully made all updates and now fetching more details for each food item, its ingredients, and description. Thanks for the review and for making our app looks more appealing when showing more details. 🤝

**Review by D4nielJ — APPROVED** [review]
## Hello @AmaduKamara and @Debas-31, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**